terraform-plans/README.md

```diff
@@ -0,0 +1,48 @@
+# Terraform plans for Solution Engineering
+
+Currently these plans are only for setting GitHub repos and to add workflow files.
+
+## How to start
```

Comment:
**suggestion (non-blocking):** 

The document here doesn't look like what we need.

I expect some document to explain the whole idea of using terraform and basic idea of how the workflow will be triggered and maintained.

Reply:
yes, I did not provide much since this was just POC. I'll update the document after reviews.

Reply:
I do not think that I will have time to do this, so we can do it in separate PR or someone else can do it here.

Reply:
This one will handover to me. Open #16 to trace.